### ISSUE TITLE
Need: ~"User Agents MAY ignore user-triggered changes to live regions, such as typing in text fields."

### ISSUE THREAD

**Issue by cookiecrook** [w3c/aria#72]
We've gotten a bug report against WebKit that VO repeats every character of a text field on Facebook.com. After investigating, the text field is placed inside an ARIA live region.

The spec ~"every change that affects the accessible label" should trigger a live region alert, but I think there should be an exception for user triggered changed like typing. 

This is probably an single-sentence "User Agents MAY" change.

**Comment by jnurthen** [comment]
moving to 1.4 and grouping all live region updates

**Comment by adampage** [comment]
Hi @cookiecrook , I’m 7 years late to this party 😅 but am wondering if you might be able to point me to the WebKit bug report that kicked off this issue so I can get the full background and see any follow-up activity?

In the meantime, I’ll review all `aria-live`-related statements in the 1.3 draft to see where we could integrate this change.

**Comment by cookiecrook** [comment]
> wondering if you might be able to point me to the WebKit bug report that kicked off this issue so I can get the full background and see any follow-up activity?

Oh gosh! I feel like the search may be an exercise in futility. I wasn’t the originator, and I didn’t store the title or tracking number in this issue. I looked briefly by keywords, but didn’t find anything obvious, and don’t feel the need to dig further.

I expect we fixed that in WebKit at the time. If it’s obviously a bug implementers don’t usually wait on “permission” from the spec.
